apps/mobile/src/libs/components/background-wrapper/libs/maps/gradient-direction-map.ts

```diff
@@ -0,0 +1,7 @@
+const gradientDirectionMap = {
```

Comment:
Map naming format should be `xToY` - in your case `directionToGradient`. Same for others